server/emailsending.js

```diff
@@ -40,7 +40,7 @@ app.post("/send", (req, res) => {
     });
 });
 
-const port =5000;
+const port =4000;
```

Comment:
Do we need to change the email port here?

Reply:
I changed it because for some reason port 5000 doesn't work with my laptop